### DISCUSSION TITLE
ESLint does not clean duplicated named exports

### DISCUSSION THREAD

**Discussion by ghost** [eslint/eslint#16925] [Q&A]
### Environment

Node version: v16.15.0
npm version: v8.5.5
Local ESLint version: v8.28.0 (Currently used)
Global ESLint version: Not found
Operating System: win32 10.0.19044

### What parser are you using?

@typescript-eslint/parser

### What did you do?

<details>
<summary>Configuration</summary>

```
<!-- Paste your code here -->
```
</details>

```js
import { useState } from 'react';
import { useState } from 'react';
```

### What did you expect to happen?

I was expecting `"import/no-duplicates": ["error"],` ESLint rule to remove duplicates, it didn't

### What actually happened?

Imports were reordered, but imports were left duplicated hanging in there

```js
import { useState, useState } from 'react';
```

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

**Demo:**
![animation](https://user-images.githubusercontent.com/86713661/220839478-8e9c51ef-5ae8-4189-b96b-0a2cf65ce768.gif)

**Comment by Rec0iL99** [comment]
Hi @alexander-vashchuk, `import/no-duplicates` is a rule from the [eslint-plugin-import](https://github.com/import-js/eslint-plugin-import) plugin (refer [docs](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-duplicates.md)). Unfortunately, we won't be able to help you here because `eslint-plugin-import` isn't a plugin that we maintain. Opening an issue there should help you. Thanks for using ESLint.
